package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.2.1",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "./build-meta.bash && vite",
```

Comment:
Actually `dev` does not need this feature, because all pages are hot reloaded.
`build` and the actual release needs this to remind people to use Ctrl-R to manual reload the page if needed.

Reply:
> Actually `dev` does not need this feature, because all pages are hot reloaded. `build` and the actual release needs this to remind people to use Ctrl-R to manual reload the page if needed.

Yes, it's hot reloaded. ~~This makes us more confident.~~